.gitignore

```diff
@@ -2,3 +2,4 @@
 .next
 node_modules
 .DS_Store
+package-lock.json
```

Comment:
You should be using Yarn to install the packages—there might be issues with deployment since Vercel will read the yarn.lock file

Reply:
ya.. umm I tried using yarn but it didn't seem to work with the @hackclub/icons package

```
sam@Sams-MacBook-Air summer-site % yarn add @next/mdx @mdx-js/loader
yarn add v1.22.4
[1/4] 🔍  Resolving packages...
[2/4] 🚚  Fetching packages...
error An unexpected error occurred: "https://registry.yarnpkg.com/@hackclub/icons/-/icons-0.0.3.tgz: Request failed \"404 Not Found\"".
info If you think this is a bug, please open a bug report with the information provided in "/Users/sam/Documents/GitHub/summer-site/yarn-error.log".
info Visit https://yarnpkg.com/en/docs/cli/add for documentation about this command.
sam@Sams-MacBook-Air summer-site % yarn add @next/mdx @mdx-js/loader
```